src/components/uielements/CasePreview.astro

```diff
@@ -1,133 +1,111 @@
+          small ? "max-h-[400px]" : "md:w-1/2 md:rounded-tr-none md:rounded-l-xl"
+        ]}>
+      <SanityImg
+          src={caseData?.introImage}
+          alt={caseData?.introImage?.asset?.altTexts?.[language] || ""}
+          widths={[400, 600, 790]}
+          sizes="(max-width: 790px) 100vw, 790px"
+          class:list={["motion-safe:group-hover:scale-105 transition-transform ease-in-out duration-300 min-w-full min-h-full max-w-[unset] h-[unset] w-auto",
+            small ? "object-scale-down shrink-0" : ""]}
+          loading="lazy"
+      />
+    </div>
+  }
+  <div class:list={["pb-8 max-h-full w-full flex flex-col justify-between gap-8 border-b-2 border-y-quaternary",
+    small ? "h-3/4 md:px-4" : "px-4 md:w-1/2 md:p-8 md:border-y-2"
+  ]}>
     <div class="flex flex-col gap-2">
       <div class="mb-4 flex flex-row items-start justify-between gap-8">
         <div class="flex flex-col">
-          <span class="font-serif text-primary" set:html={caseData.client} />
+          <span class="font-serif text-primary" set:html={caseData?.metadata?.title} />
           <h2
             class="caseheadline mb-2 font-serif text-xl [&>mark]:bg-none [&>mark]:bg-clip-border [&>mark]:text-black"
-            set:html={caseData.title}
+            set:html={caseData?.intro?.title}
           />
         </div>
         {
-          caseData.logo && (
-            <Image
+          caseData?.metadata?.image && (
+            <SanityImg
               loading="lazy"
-              src={logoimages[caseData.logo]()}
+              src={caseData?.metadata?.image?.asset?.url as SanityImageSource}
               class="h-12 w-auto shrink-0 sm:h-16 md:h-12 xl:h-16"
-              loading="lazy"
               height={60}
-              alt={caseData.client || caseData.title}
+              alt={caseData?.metadata?.image?.asset?.altTexts?.[language] || ""}
             />
           )
         }
       </div>
-      <p
-        class="text-sm font-light [&>mark]:text-black"
-        set:html={caseData.description}
-      />
+      {      
+        caseData.intro?.intro && (
+          <div class="text-sm font-light [&>mark]:text-black">
+            <PortableText value={caseData.intro?.intro} />
+          </div>
+        )
+      }
       <ol class="mt-2 list-inside list-disc font-serif text-sm">
         {
-          caseData.duration && (
+          caseData.duration != null && (
             <li class="">
               {t("metadata.duration") + ": " + caseData.duration}
             </li>
           )
         }
         {
-          caseData.collaboration && (
+          caseData.collaborationModel && (
             <li class="">
-              {t("metadata.collaboration") + ": " + caseData.collaboration}
+              {t("metadata.collaboration") + ": " + caseData.collaborationModel.collaborationTabs.map(tab => tab?.concept?.prefLabel).join(', ')}
```

Comment:
this should probably be caseData?.collaborationModel?.collaborationTabs?.map just to be safe and that it won't crash in live edit mode.